composer.json

```diff
@@ -15,8 +15,8 @@
         "psr-4": { "M6Web\\Component\\": "src/" }
     },
     "require": {
-        "php"                         : ">=5.4.0",
-        "elasticsearch/elasticsearch" : "^2.0"
+        "php"                         : ">=5.6.6",
```

Comment:
I think we could set this to `^7.0` because it's what they've done on the master branch of the `elasticsearch/elasticsearch` project ( https://github.com/elastic/elasticsearch-php/blob/master/composer.json#L13 )

but they didn't in their `5.0` branche yet, so maybe it's better to keep 5.6.6 👍